### ISSUE TITLE
Styles fail to install if the server doesn't sent "text/css" as content type

### ISSUE THREAD

**Issue by NiklasGollenstede** [NiklasGollenstede/re-style#1]
This causes: 
> when installing one, I still get an error (even though it will get listed to my styles):
`SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data`

https://github.com/NiklasGollenstede/native-ext/issues/2#issuecomment-340141459

**Comment by NiklasGollenstede** [comment]
> I tried [these styles](https://github.com/Timvde/UserChrome-Tweaks), by the way.

I have't looked through all of them, but as it says in the repo description, they are userChrome snippets, not stylish styles. reStyle is meant as a replacement for Stylish and requires `@document` sections to know where to put the different kinds of styles. Without them, the only thing reStyle could do would be to apply the style to the UI, `about:`-pages and every web page. That is usually not what you actually want and would be very inefficient. Most of the styles in the linked repo should actually be surrounded with this (even when directly pasted to the `userChrome.css` file):

```
@-moz-document
    url(chrome://browser/content/browser.xul)
{
    /* css from file here */
}
```
Without a surrounding `@document`  rule (or a `@namespace url("http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul");`), reSytle will not recognize a style as a UI style.

**Comment by NiklasGollenstede** [comment]
The problem itself is fixed in 89ac2e1cbbfc49c86ecae3378b2eb02abf02df89. I'll update the beta on AMO.
